README.md

```diff
@@ -333,6 +334,8 @@ elasticity, but uses roughly half as many IPs as using WARM_IP_TARGET alone (32
 This also improves the reliability of the EKS cluster by reducing the number of calls necessary to allocate or deallocate
 private IPs, which may be throttled, especially at scaling-related times.
 
+**NOTE!** If MINIMUM_IP_TARGET is set WARM_ENI_TARGET will be ignored, please utilize WARM_IP_TARGET instead. Please also referring `WARM_ENI_TARGET`.
```

Comment:
format nit: "**NOTE!** If `MINIMUM_IP_TARGET` is set, `WARM_ENI_TARGET` will be ignored. Please utilize `WARM_IP_TARGET` instead. Please also refer to `WARM_ENI_TARGET`.

Reply:
Nit: `referring` -> `refer`

I don't think we need -> Please also refer to WARM_ENI_TARGET...

Reply:
Let's also add that if `MINIMUM_IP_TARGET` is set, but `WARM_IP_TARGET` is not set, `WARM_IP_TARGET` is assumed to be 0, which will prevent future ENIs from being allocated. We should strongly recommend that `WARM_IP_TARGET` is configured when `MINIMUM_IP_TARGET` is

Reply:
Updated the content, can you check again? Thanks.